tutorials/integration.qmd

```diff
@@ -0,0 +1,171 @@
+---
+title: Integration
+---
+
+```{julia}
+#| echo: false
+#| code-fold: true
+#| output: false
+using Pkg;
+cd(@__DIR__)
+Pkg.activate("."); # for reproducibility use the local tutorial environment.
+Pkg.develop(PackageSpec(; path=(@__DIR__) * "/../"))
+using Markdown
+```
+
+This part of documentation covers integration of scalar functions defined on manifolds.
+The basic concepts are derived from geometric measure theory.
+In principle, there are many ways in which a manifold can be equipped with a measure that can be later used to define an integral.
+One of the most popular ways is based on pushing the Lebesgue measure on a tangent space through the exponential map.
+Any other suitable atlas could be used, not just the one defined by normal coordinates, though each one requires different volume density corrections due to the Jacobian determinant of the pushforward.
+`Manifolds.jl` provides the function ``[`volume_density`](@ref)``{=commonmark} that calculates that quantity, denoted $\theta_p(X)$.
+See for example [^leBrigantPuechmorel2019], Definition 11, for a precise description using Jacobi fields.
```

Comment:
I've tried the commonmark thing but IMO on its own it makes things worse: bibliography is still numbered instead of by key. `[1]` seems perfectly usable here and I'd prefer to leave to you updating bibliography here similarly to Manopt.

Reply:
I fear I am too lazy to update your bibliography ;) 

My only sad point is that then this page is very different from the rest of the docs, but still in style close (with the footnotes).

Reply:
Yes, it doesn't look great but I hope maybe at some point there will be an easier way to manage bibliography so I'm not particularly motivated to fight with it now.

Reply:
Ys a proper way to handle bibliography in Documenter would be very welcome. Ok let's keep it like that for now.